### PR TITLE
Leisure

### PR DESCRIPTION
**Pull request by BryceHaley**
Ready for review.

### CODE REVIEW

File: leisure/leisure.ipynb [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Callysto’s Weekly Data Visualization\n",
+    "\n",
+    "## Weekly Title\n",
+    "\n",
+    "### Recommended grade level: 6-12"
```
Comment (TinaL5) [inline leisure/leisure.ipynb:18]: Recommended Grade level: 5-12

File: leisure/leisure.ipynb [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    "fig1.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interpret-1\n",
+    "\n",
+    "Our first figure shows 15-24 year olds spending a lot more time than their older counterparts playing computer games but with a small bump for Canadians in early retirement age(55-64 and 65-75).\n",
```
Comment (TinaL5) [inline leisure/leisure.ipynb:150]: space after age

File: leisure/leisure.ipynb [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    "import plotly.express as px #used to create interactive plots"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The code below creates lists data from [this 2010 StatsCan table](https://www150.statcan.gc.ca/n1/pub/89-647-x/2011001/tbl/tbl31-eng.html). The same study was done more recently in 2015. However, the more recent time use survey did not ask about video games.\n",
+    "\n",
+    "Our lists our as follows:\n",
```
Comment (TinaL5) [inline leisure/leisure.ipynb:91]: are

**Review by TinaL5 — APPROVED** [review]

File: leisure/leisure.ipynb (hunk 3 above)
Comment (BryceHaley) [reply leisure/leisure.ipynb:91]: One day my monolingual self will master this one language.

**Comment by TinaL5** [comment]
@BryceHaley Just noticed that the StatsCan link is broken.

**Review by TinaL5 — CHANGES_REQUESTED** [review]
This link doesn't still doesn't work for me: https://www150.statcan.gc.ca/n1/pub/89-647-x/2011001/tbl/tbl31-eng.html

**Review by TinaL5 — APPROVED** [review]

**Comment by BryceHaley** [comment]
@TinaL5 link was replaced with: https://www150.statcan.gc.ca/n1/pub/89-647-x/2011001/tbl/tbl31-eng.html which should work - although my eye can't see the difference between the two.

**Pull request merged**